mautic/mautic
# Focus Custom CSS

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Y
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added custom CSS to focus integration

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create modal window with custom style `.mf-modal { border: 10px solid #5D0E4E }`
3. You should see something like this

![image](https://user-images.githubusercontent.com/462477/83003590-d0208580-a00e-11ea-9a6a-591f2bc7c555.png)

4. Test also with inline script on your page


<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/8841"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
